crates/core/src/db/models.rs

```diff
@@ -3376,3 +3376,41 @@ pub struct ProposalTransactionInstructionAccount<'a> {
     /// The write version of this account's last known update
     pub write_version: i64,
 }
+
+#[derive(Debug, Clone, Queryable, Insertable, AsChangeset)]
+#[diesel(treat_none_as_null = true)]
+#[allow(missing_docs)]
+pub struct Collection<'a> {
+    pub id: Cow<'a, str>,
+    pub image: Cow<'a, str>,
+    pub name: Cow<'a, str>,
+    pub description: Cow<'a, str>,
+    pub verified_collection_address: Option<Cow<'a, str>>,
+    pub pieces: i64,
+    pub verified: bool,
+    pub created_at: NaiveDateTime,
+    pub updated_at: NaiveDateTime,
```

Comment:
There are some additional fields that I'd like to index. Can you include this set please?

```
id [primary key]
image 
name
description
twitter_url
discord_url
website_url
magic_eden_identifier [index]
verified
pieces
go_live_at
verified_collection_address
created_at
updated_at
```